drake/systems/plants/RigidBodyTree.cpp

```diff
@@ -1037,28 +1036,21 @@ Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic> RigidBodyTree::massMatrix(Kinemat
   return ret;
 }
 
-/**
- * note that this method can also be used to compute the gravitational term only by calling doKinematics with a zero joint velocity vector.
- * To compute only the Coriolis term, pass in nullptr for vd and set gravity to zero.
- * Note that the wrenches in f_ext are expressed in body frame.
- */
```

Comment:
Moved to header file. It's the right place for documentation anyway, but putting documentation here also caused there to be a separate entry for the implementation in the documentation because the implementation doesn't specify the default arguments.